patch/kernel/rk35xx-vendor-6.1/drm_log_flood.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c b/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
+index 12d8d96c8727..2dc37c3a6dfe 100644
+--- a/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
++++ b/drivers/gpu/drm/rockchip/rockchip_drm_vop2.c
+@@ -2583,7 +2583,7 @@ static bool rockchip_vop2_mod_supported(struct drm_plane *plane, u32 format, u64
+ 	if (!rockchip_afbc(plane, modifier) &&
+ 	    !rockchip_rfbc(plane, modifier) &&
+ 	    !rockchip_tiled(plane, modifier)) {
+-		DRM_ERROR("%s unsupported format modifier 0x%llx\n", plane->name, modifier);
++		//DRM_ERROR("%s unsupported format modifier 0x%llx\n", plane->name, modifier);
```

Comment:
It is more appropriate to adjust the drm log level, these logs are useful for developers.
```suggestion
+		DRM_DEBUG("%s unsupported format modifier 0x%llx\n", plane->name, modifier);
```